pyproject.toml

```
@@ -117,3 +121,6 @@ check_untyped_defs = true
 [[tool.mypy.overrides]]
 module = [ "cftime.*", "scipy.*", "fsspec.*", "perceiver_pytorch.*" ]
 follow_untyped_imports = true
+
+[tool.uv]
+no-build-isolation-package = [ 'flash-attn' ]
```

Rather than turning of build isolation completely, I suggest we do what the [uv docs give as an example for this package](https://docs.astral.sh/uv/concepts/projects/config/#augmenting-build-dependencies)

```suggestion
[tool.uv.extra-build-dependencies]
flash-attn = [{ requirement = "torch", match-runtime = true }]

[tool.uv.extra-build-variables]
flash-attn = { FLASH_ATTENTION_SKIP_CUDA_BUILD = "TRUE" }
```